packages/core/src/tools/stackTrace/computeStackTrace.spec.ts

```diff
@@ -1012,4 +870,65 @@ Error: foo
       url: 'capacitor://localhost/media/dist/bundle.js',
     })
   })
+
+  it('should parse Safari WebAssembly stack frames', () => {
+    const wasmStack = `
+  RuntimeError: Out of bounds memory access
+      myModule.wasm-function[crashHandler]@[wasm code]
+      handleError@https://example.com/app.js:123:45
+      @https://example.com/app.js:150:30
+    `
+    const mockErr = { message: 'Out of bounds memory access', name: 'RuntimeError', stack: wasmStack }
+
+    const stackFrames = computeStackTrace(mockErr)
+
+    expect(stackFrames.stack.length).toBe(3)
+
+    expect(stackFrames.stack[0]).toEqual({
+      func: 'myModule.wasm-function[crashHandler]',
+      url: '[wasm code]',
+      column: undefined,
+      line: undefined,
+      args: [],
+    })
+  })
+
+  it('should parse Chrome WebAssembly stack frames', () => {
+    const wasmStack = `
+  Error: Wasm Error
+    at foo (<anonymous>:wasm-function[42]:0x1a3b)
+    at bar (http://example.com/script.js:10:5)
+    `
+    const mockErr = { message: 'Wasm Error', name: 'Error', stack: wasmStack }
+    const stackFrames = computeStackTrace(mockErr)
+
+    expect(stackFrames.stack.length).toBe(2)
+    expect(stackFrames.stack[0]).toEqual({
+      args: [],
+      func: 'foo',
+      url: '<anonymous>:wasm-function[42]:0x1a3b',
+      line: undefined,
+      column: undefined,
+    })
+  })
+
+  it('should parse Firefox WebAssembly stack frames', () => {
+    const wasmStack = `
+  Error: Wasm Error
+    wasm-function[42]@http://example.com/my-module.wasm:wasm-function[42]:0x1a3b
+    bar@http://example.com/script.js:10:5
+    `
+    const mockErr = { message: 'Wasm Error', name: 'Error', stack: wasmStack }
+    const stackFrames = computeStackTrace(mockErr)
+
+    expect(stackFrames.stack.length).toBe(2)
+
+    expect(stackFrames.stack[0]).toEqual({
+      args: [],
+      func: 'wasm-function[42]',
+      url: 'http://example.com/my-module.wasm:wasm-function[42]:0x1a3b',
```

Comment:
This is a mis-parse; the URL here should just be:
```suggestion
      url: 'http://example.com/my-module.wasm',
```